web-server/v0.4/src/models/explore.js

```diff
@@ -11,19 +11,15 @@ export default {
   effects: {
     *fetchSharedSessions({ payload }, { call, put }) {
       const response = yield call(querySharedSessions, payload);
-
+      console.log(response);
```

Comment:
Why do we need this log entry on the console?

Reply:
Fixed. Left it by mistake.